pkg/3scale/amp/template/ampevals3.go

```diff
@@ -10,23 +11,31 @@ func init() {
 	TemplateFactories = append(TemplateFactories, NewAmpEvalS3TemplateFactory)
 }
 
-type EvalS3Adapter struct {
+type AmpEvalS3TemplateAdapter struct {
 }
 
-func (e *EvalS3Adapter) Adapt(template *templatev1.Template) {
+func (e *AmpEvalS3TemplateAdapter) Adapt(template *templatev1.Template) {
 	template.Name = "3scale-api-management-eval-s3"
-	template.ObjectMeta.Annotations["description"] = "3scale API Management main system (Evaluation) with shared file storage in AWS S3."
+	template.Annotations["description"] = "3scale API Management main system (Evaluation) with shared file storage in AWS S3."
 }
 
 type AmpEvalS3TemplateFactory struct {
 }
 
 func (f *AmpEvalS3TemplateFactory) Adapters() []adapters.Adapter {
-	ampFactory := NewAmpTemplateFactory()
-	evalAdapter := adapters.NewEvalAdapter()
-	s3Adapter := adapters.NewS3Adapter()
```

Comment:
As you can see, we've removed S3 component completely from templates and operator